### ISSUE TITLE
Ancestors and Descendants charts - Individuals and Families - 2.0.0

### ISSUE THREAD

**Issue by melizaa** [fisharebest/webtrees#1606]
I miss the zebra lines and table column and row grids. 
We see
![2018-02-08_10-36-23](https://user-images.githubusercontent.com/5202996/35962970-f4a5be0e-0cbb-11e8-91a5-8e8c3f795955.jpg)
We used to see
![2018-02-08_10-39-01](https://user-images.githubusercontent.com/5202996/35963100-535285e0-0cbc-11e8-847a-85ff1c1ba2ee.jpg)

We see Show parents and Show statistics charts buttons - they do not work.

**Comment by melizaa** [comment]
The top filter buttons do not work.

**Comment by melizaa** [comment]
Sorting the data does not work.

**Comment by fisharebest** [comment]
We are missing the javascript to enable the interactive table functions.

**Comment by melizaa** [comment]
Same issue in the source, media and shared notes pages.

index.php?route=source&xref=Snn&ged=, index.php?route=media&xref=Mnn&ged= and index.php?route=note&xref=Nnn&ged=

**Comment by melizaa** [comment]
We can no longer see a partial list

![2018-03-10_22-31-36](https://user-images.githubusercontent.com/5202996/37246518-df655572-24b2-11e8-8c35-d76591e58212.jpg)

**Comment by melizaa** [comment]
The Ancestry lists seem now to work.

I played with the Descendants list - it does not yet work - I saw
Fatal error: Uncaught TypeError: Argument 1 passed to Fisharebest\Webtrees\Individual::getInstance() must be of the type string, null given, called in /home/amitys/test.amitys.com/app/Http/Controllers/DescendantsChartController.php on line 69 and defined in /home/amitys/test.amitys.com/app/Individual.php:53 Stack trace: #0 /home/amitys/test.amitys.com/app/Http/Controllers/DescendantsChartController.php(69): Fisharebest\Webtrees\Individual::getInstance(NULL, Object(Fisharebest\Webtrees\Tree)) #1 [internal function]: Fisharebest\Webtrees\Http\Controllers\DescendantsChartController->page(Object(Symfony\Component\HttpFoundation\Request), Object(Fisharebest\Webtrees\Tree)) #2 /home/amitys/test.amitys.com/app/Resolver.php(88): ReflectionMethod->invoke(Object(Fisharebest\Webtrees\Http\Controllers\DescendantsChartController), Object(Symfony\Component\HttpFoundation\Request), Object(Fisharebest\Webtrees\Tree)) #3 /home/amitys/test.amitys.com/index.php(279): Fisharebest\Webtrees\Resolver->dispatch(Object(Fisharebest\Webtrees\Http\C in /home/amitys/test.amitys.com/app/Individual.php on line 53

**Comment by DutchyAlpaca** [comment]
I had a look at this, new to webtrees but I think I found something.

The problem is the views/lists/individuals-table.php etc.

On line 17: View::push.
This only works on the search page and no other because of the ajax_load through descendants page for example.

I haven't thought of a nice way to fix it.

But for now I have copied the file and removed line 17 and 73 in the original.

Renamed the copy: Search-individuals-table.php

And then changed what view to use on line 65 in views/search-results.php

I Hope someone can think of an if statement around line 17 and 73 to only do the view::push when it's used on the search results page.
